nginx.rb

```diff
@@ -10,18 +10,21 @@ class Nginx < FPM::Cookery::Recipe
 
   section 'System Environment/Daemons'
 
-  build_depends 'make', 'gcc', 'g++', 'libssl-dev', 'libxml2-dev', 'libxslt1-dev'
```

Comment:
why no `libssl-dev`?

Reply:
Because we build Ruby 2.1.2 for the bundled applications and pre 2.4 Ruby works with libssl 0.1 only. New release of Ubuntu 18.04 has newer version provided as libssl and libssl-dev and legacy provided as libssl0.1.0 and libssl0.1-dev. This is why we can't have same build dependencies for, say, xenial and bionic - same package has different names.